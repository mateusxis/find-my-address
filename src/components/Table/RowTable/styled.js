import styled from 'styled-components';

export const SRow = styled.tr`
  background-color: #ffffff;
  box-shadow: 0px 0px 9px 0px rgba(0, 0, 0, 0.1);
  border-radius: 3px;
  padding: 15px 20px;
  justify-content: space-between;
  display: block;
  margin-bottom: 25px;

  @media (min-width: 961px) {
    display: flex;
  }
`;
