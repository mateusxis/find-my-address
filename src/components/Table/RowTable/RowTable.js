import { SRow } from './styled';

const RowTable = ({ children }) => {
  return <SRow>{children}</SRow>;
};

export default RowTable;
