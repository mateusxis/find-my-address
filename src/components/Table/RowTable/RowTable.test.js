import renderer from 'react-test-renderer';

import RowTable from './RowTable';

describe('RowTable', () => {
  it('should render RowTable', () => {
    const tree = renderer
      .create(
        <RowTable>
          <td data-label="City" className="c1">
            New York City
          </td>
        </RowTable>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
