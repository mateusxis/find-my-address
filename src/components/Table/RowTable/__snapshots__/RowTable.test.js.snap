// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RowTable should render RowTable 1`] = `
<tr
  className="sc-bdvvtL bgbyWY"
>
  <td
    className="c1"
    data-label="City"
  >
    New York City
  </td>
</tr>
`;
